clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/UpsertAmazonLoadBalancerClassicDescription.java

```diff
@@ -14,35 +14,30 @@
  * limitations under the License.
  */
 
-package com.netflix.spinnaker.clouddriver.aws.deploy.description
+package com.netflix.spinnaker.clouddriver.aws.deploy.description;
 
-class UpsertAmazonLoadBalancerDescription extends AbstractAmazonCredentialsDescription {
-  String clusterName
-  String name
-  String vpcId
-  Boolean isInternal
-  String subnetType
-  List<String> securityGroups
-  Map<String, List<String>> availabilityZones
-  List<Listener> listeners
-  String healthCheck
-  Integer healthInterval = 10
-  Integer healthTimeout = 5
-  Integer unhealthyThreshold = 2
-  Integer healthyThreshold = 10
-  Boolean crossZoneBalancing = Boolean.TRUE
+import java.util.List;
 
-  static class Listener {
-    enum ListenerType {
+public class UpsertAmazonLoadBalancerClassicDescription extends UpsertAmazonLoadBalancerDescription {
+  public List<Listener> listeners;
```

Comment:
these should be private with getter/setter (what the equivalent groovy was generating).